grafana/alternator.master.template.json

```diff
@@ -942,6 +1112,29 @@
                         }
                     ],
                     "query": "GetItem,PutItem,UpdateItem,DeleteItem"
+                },
```

Comment:
I'm not sure it does what you want it to do. This is a template variable with a single value, if you only have one static value (GetRecords), no need to make it a variable.

If there are multiple values you want to support, they are missing.

If this is a preparation for future extension, maybe it's premature, but ok

Reply:
I was trying to follow the pattern that was there before, to minimize my work. 
not that I know of more APIs we'll need have latency metric for, again this is still early to tell which would be mostly needed and if we'll add more metrics to the APIs.